src/page-modules/contact/means-of-transport/forms/delayForm.tsx

```diff
@@ -44,8 +43,8 @@ export const DelayForm = ({ state, send }: DelayFormProps) => {
         </Typo.p>
 
         <Select
-          label={t(PageText.Contact.input.transportMode.label).toString()}
-          value={state.context.transportMode}
+          label={t(PageText.Contact.input.transportMode.label)}
```

Comment:
Seems like we could've benefitted from having a separate component for transport mode selection seeing how many of these was updated in this PR.

Reply:
Agree. Maybe I should have a look at it in a separate PR?